docs/source/devops.rst

```diff
@@ -21,3 +21,24 @@ Example:
 ::
 
   universal-login deploy:token --nodeUrl http://localhost:18545 --privateKey 0x29f3edee0ad3abf8e2699402e0e28cd6492c9be7eaab00d732a791c33552f797
+
+
+Sending funds
+-------------
+
+To send funds to an address
+``universal-login send [to] [amount] [currency] --nodeUrl [url] --privateKey [privateKey]``
+
+Parameters: 
+  - **to** - an address to send funds
+  - **amount** - amount to send to the address
+  - **currency** - currency of transfer
+  - **nodeUrl** (optional) - JSON-RPC URL of an Ethereum node, set to ``http://localhost:18545`` by default
+  - **privateKey** (optional) - private key of wallet with additional balance, set to ``DEV_DEFAULT_PRIVATE_KEY`` by default and has enoguh ethers
```

Comment:
here we could explain it in plain words, e.g. 
by default it will be set to a wallet that have some ether on development environment